job/validators.py

```diff
@@ -352,24 +354,17 @@ def cross_clean_pv(pvmodel):
         ElectricLoad
         If user does not provide values, set defaults conditional on off-grid flag
         """
-
-        if self.models["ElectricLoad"].__getattribute__("critical_load_pct") == None:
-            if self.models["Settings"].off_grid_flag==False:
-                self.models["ElectricLoad"].critical_load_pct = 0.5
-            else:
+        '''
+        If off-grid scenario, then always set critical load pct to 1.0, overriding any other value the user might have provided.
+        If !off-grid scenario, if critical load pct is not specified in inputs, set it to 0.5. Otherwise, allow user specified pct.
+        '''
+        if self.models["Settings"].off_grid_flag==True:
                 self.models["ElectricLoad"].critical_load_pct = 1.0
-        
-        if self.models["ElectricLoad"].__getattribute__("operating_reserve_required_pct") == None:
-            if self.models["Settings"].off_grid_flag==False:
-                self.models["ElectricLoad"].operating_reserve_required_pct = 0.0
-            else:
                 self.models["ElectricLoad"].operating_reserve_required_pct = 0.1
-
-        if self.models["ElectricLoad"].__getattribute__("min_load_met_annual_pct") == None:
-            if self.models["Settings"].off_grid_flag==False:
-                self.models["ElectricLoad"].min_load_met_annual_pct = 1.0
-            else:
                 self.models["ElectricLoad"].min_load_met_annual_pct = 0.99999
```

Comment:
@rathod-b Similar to above comment, `min_load_met_annual_pct` should not get overwritten if `off_grid_flag=True`. Please change validators.py to reflect this.

Reply:
Addressed in https://github.com/NREL/REopt_API/pull/343/commits/152ac98842eaaf16a7068f0412b34b0893f0886a